### ISSUE TITLE
Nuxt does not register Vue instance injection typings of modules.

### ISSUE THREAD

**Issue by komninoschat** [nuxt/typescript#128]
**Describe the bug**
Nuxt does not register Vue instance injection typings of modules.

**To Reproduce**
1. Create a new Vue project with https://github.com/nuxt-community/typescript-template which already has `@nuxtjs/axios` installed
2. Try to access `this.$axios` inside a Vue instance
3. nuxt-ts will emit a warning that $axios does not exist.

**Expected behavior**
Nuxt-ts should not emit any warnings.

**Screenshots**
![image](https://user-images.githubusercontent.com/29049104/65540486-30a57300-df14-11e9-868a-cdcfe0b07a54.png)
![image](https://user-images.githubusercontent.com/29049104/65540509-3ac77180-df14-11e9-83de-d57f0e42e90c.png)

**Additional context**
As you can see `@nuxtjs/axios` injects into the Vue instance so it shouldn't be a problem with the library. I have the same issue with a TS module I'm developing for nuxt.
https://github.com/nuxt-community/axios-module/blob/dev/types/index.d.ts#L44

**Comment by rchl** [comment]
You need to manually add all the types you need in `types` array in `tsconfig.json`. Here is an example of a simple setup - https://typescript.nuxtjs.org/guide/setup.html#installation . The page is not super explicit about it though. Maybe that could be improved.

**Comment by kevinmarrec** [comment]
```json
"compilerOptions": {
  "types": [
    "@nuxt/types",
    "@nuxtjs/axios"
    "@types/node"
  ]
}
```

**Comment by komninoschat** [comment]
Okay. Turns out I just had to add it to the types list of my Typescript config file. Thanks @rchl and @kevinmarrec
